examples/step-18/step-18.cc

```diff
@@ -1826,42 +1810,13 @@ namespace Step18
 
 int main(int argc, char **argv)
 {
-  try
-    {
-      using namespace dealii;
-      using namespace Step18;
+  using namespace dealii;
+  using namespace Step18;
 
-      Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, 1);
+  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, 1);
 
-      TopLevel<3> elastic_problem;
-      elastic_problem.run();
-    }
-  catch (std::exception &exc)
-    {
-      std::cerr << std::endl
-                << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-      std::cerr << "Exception on processing: " << std::endl
-                << exc.what() << std::endl
-                << "Aborting!" << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-
-      return 1;
-    }
-  catch (...)
-    {
-      std::cerr << std::endl
-                << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-      std::cerr << "Unknown exception!" << std::endl
-                << "Aborting!" << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-      return 1;
-    }
+  TopLevel<3> elastic_problem;
+  elastic_problem.run();
```

Comment:
From #6784: we should keep the handler so that MSVC can print error messages.

Reply:
@drwells done.